.github/workflows/docker.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     tags:
       - v*
-    branches: [chain4travel, dev]
   workflow_dispatch:
     inputs:
       wd_branch:
```

Comment:
why do we need this ? why not to get the branch from github variables like (GITHUB_REF_NAME)

Reply:
We don't need it, because of this it is removed....

Reply:
Will remove inputs